analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.CSharp9.cs

```diff
@@ -66,16 +66,25 @@ public void PatternMatching(int? arg)
             {
                 v = nullable.Value;
             }
-            v = nullable.Value;        // FN, can't build CFG for this method
+            v = nullable.Value;        // Noncompliant
         }
     }
 
+    void PatternMatching(int? i, double? d, float? f)
+    {
+        if (i.HasValue is true && i.Value == 42) { }
+        if (i is not null && i.Value == 42) { }
+
+        if (i.HasValue is not true && i.Value == 42) { } // Noncompliant
+        if (i is null && i.Value == 42) { } // Noncompliant
```

Comment:
This would read easier when aligned to the same tab distance
```suggestion
        if (i.HasValue is not true && i.Value == 42) { } // Noncompliant
        if (i is null && i.Value == 42) { }              // Noncompliant
```